_posts/2014-04-07-omero-bio-formats-5-0-1.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Release of OMERO & Bio-Formats 5.0.1
+intro-blurb: The OME team are pleased to announce the release of OMERO & Bio-Formats 5.0.1
+---
+Today we are releasing OMERO and Bio-Formats 5.0.1. This a bug-fixing
+release covering a number of issues as listed below.
+
+Our plan for the 5.0.x line is to do a series of these bug-fixing
+releases. Therefore, if you are having an issue with 5.0.0 which has
+not yet been addressed, be assured that 5.0.2 will be following with
+further fixes as soon as possible. Note that only the jar-signing
+issue will be back-ported to the 4.4.x line - we will be releasing
+4.4.11 imminently with this fix included.
+
+For OMERO, this release includes:
+
+-  Code signing to fix the Java Web Start issues
+-  New import scenarios [documentation](http://www.openmicroscopy.org/site/support/omero5/sysadmins/index.html#data-import-options) including for the new 'in-place' import capability
```

Comment:
:+1: once this PR is merged, I can certainly use https://github.com/openmicroscopy/www.openmicroscopy.org/pull/77 to:

- update all the recent doc links to use the new URL
- review the remaining `site/support` URLs